shared/constant/constant.go

```diff
@@ -16,10 +16,10 @@ package constant
 
 // PrivilegedPlugins can be changed by 'WOODPECKER_ESCALATE' at runtime
 var PrivilegedPlugins = []string{
-	"plugins/docker",
-	"plugins/gcr",
-	"plugins/ecr",
-	"woodpeckerci/plugin-docker-buildx",
+	"docker.io/plugins/docker",
```

Comment:
Unsure about this change, this would be a breaking change as auto-escalate doesn't work anymore for image names without the prefix.

Reply:
Reverted.